### PR TITLE
Bump package versions 4.3.0 and WebUI to 4.3.0 Alpha1 and Uyuni 2021.06

### PR DESCRIPTION
**Pull request by juliogonzalez**
## What does this PR change?

Bump package versions 4.3.0 and WebUI to 4.3.0 Alpha1 and Uyuni 2021.06

We'll do it now because of two reasons:

a) I don't want tag conflicts
b) After 4.2 GM is created, it's when 4.3.X starts.
c) Bumping the version for Uyuni on top of Leap 15.3 is not a bad idea, as it's when we are bumping the Leap version.

This also passing /usr/lib/obs/service/format_spec_file as requested by autobuild.

For:
```
[WARNING] Macro for LICENSE at client/tools/mgr-cfg/mgr-cfg.spec is %doc, which is probably wrong!
[WARNING] Macro for LICENSE at client/tools/mgr-custom-info/mgr-custom-info.spec is %doc, which is probably wrong!
[WARNING] Macro for LICENSE at client/rhel/mgr-daemon/mgr-daemon.spec is %doc, which is probably wrong!
[WARNING] Macro for LICENSE at client/tools/mgr-osad/mgr-osad.spec is %doc, which is probably wrong!
[WARNING] Macro for LICENSE at client/tools/mgr-virtualization/mgr-virtualization.spec is %doc, which is probably wrong!
[WARNING] Macro for LICENSE at projects/perl-Satcon/perl-Satcon.spec is %doc, which is probably wrong!
[WARNING] Macro for LICENSE at branding/spacewalk-branding.spec is %doc, which is probably wrong!
[WARNING] Macro for LICENSE at spacewalk/config/spacewalk-config.spec is %doc, which is probably wrong!
[WARNING] Macro for LICENSE at proxy/proxy-html/spacewalk-proxy-html.spec is %doc, which is probably wrong!
[WARNING] Macro for LICENSE at proxy/installer/spacewalk-proxy-installer.spec is %doc, which is probably wrong!
[WARNING] Macro for LICENSE at spacewalk/pylint/spacewalk-pylint.spec is %doc, which is probably wrong!
[WARNING] Macro for LICENSE at spacewalk/spacewalk-python2-pylint/spacewalk-python2-pylint.spec is %doc, which is probably wrong!
[WARNING] Macro for LICENSE at spacewalk/spacewalk-python3-pylint/spacewalk-python3-pylint.spec is %doc, which is probably wrong!
[WARNING] Macro for LICENSE at spacewalk/setup/spacewalk-setup.spec is %doc, which is probably wrong!
[WARNING] Macro for LICENSE at spacewalk/spacewalk-setup-jabberd/spacewalk-setup-jabberd.spec is %doc, which is probably wrong!
[WARNING] Macro for LICENSE at spacewalk/spacewalk-setup-postgresql/spacewalk-setup-postgresql.spec is %doc, which is probably wrong!
[WARNING] Macro for LICENSE at projects/ssl-cert-check/spacewalk-ssl-cert-check.spec is %doc, which is probably wrong!
[WARNING] Macro for LICENSE at uyuni/common-libs/uyuni-common-libs.spec is %doc, which is probably wrong!
[WARNING] Macro for LICENSE at client/rhel/yum-rhn-plugin/yum-rhn-plugin.spec is %doc, which is probably wrong!
```
For those package used at SLE11/CentOS6 I replaced `%doc LICENSE` with:
```
%{!?_licensedir:%global license %doc}
%license doc/LICENSE
```
And in all other cases I replaced it with `%license LICENSE`

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: Internal changes

- [x] **DONE**

## Test coverage
- No tests: To be tested at OBS

- [x] **DONE**

## Links

Related: https://github.com/SUSE/spacewalk/issues/14997

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by jordimassaguerpla — APPROVED** [review]

**Comment by juliogonzalez** [comment]
PR tests running at https://ci.suse.de/view/Manager/view/Uyuni-PRs/job/uyuni-prs-ci-tests/

**Comment by juliogonzalez** [comment]
Seems some packages failed to build: https://build.opensuse.org/project/monitor/systemsmanagement:Uyuni:Master:PR:3851?arch_x86_64=1&defaults=0&failed=1&repo_openSUSE_Leap_15_3=1

I will have a look tomorrow, seems most of them are because of the new usage of `%license`

**Comment by jordimassaguerpla** [comment]
I just retriggered the build disabling the "remove build" option, so that packages are not cleaned up afterwards. This way we can take a look at the failures at https://build.opensuse.org/project/monitor/systemsmanagement:Uyuni:Master:PR:3851?arch_x86_64=1&defaults=0&failed=1&repo_openSUSE_Leap_15_3=1

In order to do a cleanup, please rebuild the job with the option "remove build" enabled, but the "build" and "test" disabled, so the job will do a cleanup.

**Review by mcalmer — COMMENTED** [review]
One comment you might want to consider. Also check if we need to change to dependencies defined via tito.props

File: schema/spacewalk/susemanager-schema.spec [hunk 1]
```diff
@@ -24,8 +24,8 @@ Summary:        SQL schema for Spacewalk server
 License:        GPL-2.0-only
 Group:          Applications/Internet
 
-Version:        4.2.14
-Release:        1%{?dist}
+Version:        4.3.0
+Release:        0
```
[the remark this answers is not in the document]
Comment (juliogonzalez) [reply schema/spacewalk/susemanager-schema.spec:28]: Done, and in fact we need one from 4.3.0 to 4.3.1 :-)

File: client/tools/spacewalk-abrt/spacewalk-abrt.spec [hunk 2]
```diff
@@ -29,13 +29,13 @@
 %define pythonX %{?default_py3: python3}%{!?default_py3: python2}
 
 Name:           spacewalk-abrt
-Version:        4.2.1
-Release:        1%{?dist}
+Version:        4.3.0
+Release:        0
```
[the remark this answers is not in the document]
Comment (juliogonzalez) [reply client/tools/spacewalk-abrt/spacewalk-abrt.spec:33]: We should do a cleanup. I still see oracle stuff as well. But for now, I will just do the bump.

**Comment by juliogonzalez** [comment]
> One comment you might want to consider. Also check if we need to change to dependencies defined via tito.props

Done. Hopefully the changes are OK.

**Comment by juliogonzalez** [comment]
Rerunning tests at https://ci.suse.de/view/Manager/view/Uyuni-PRs/job/uyuni-prs-ci-tests/314/

**Review by mcalmer — CHANGES_REQUESTED** [review]
Too many schema migration directories created. The first one pointing to 4.3.0 must be the one after the current 4.2.X version. Otherwise we loose all migrations in 4.2.

**Comment by juliogonzalez** [comment]
> Too many schema migration directories created. The first one pointing to 4.3.0 must be the one after the current 4.2.X version. Otherwise we loose all migrations in 4.2.

Ah, true, true. For the Uyuni users they will have all of them, and for SUSE Manager users, they first need to update to the latest 4.2.

Fixing it.

**Comment by juliogonzalez** [comment]
> > Too many schema migration directories created. The first one pointing to 4.3.0 must be the one after the current 4.2.X version. Otherwise we loose all migrations in 4.2.
> 
> Ah, true, true. For the Uyuni users they will have all of them, and for SUSE Manager users, they first need to update to the latest 4.2.
> 
> Fixing it.

Should be OK now.

**Comment by juliogonzalez** [comment]
New tests running at https://ci.suse.de/view/Manager/view/Uyuni-PRs/job/uyuni-prs-ci-tests/315/

**Review by mcalmer — APPROVED** [review]

**Comment by juliogonzalez** [comment]
Seems all packages built correctly, so we can merge this after:
- We have 4.2 GM
- We branch `Manager-4.2`
- We unlock `master`

**Comment by juliogonzalez** [comment]
All conflicts solved. This PR is to be merged before ANY other.

**Pull request merged**